server/container/util-java8/src/main/java/org/apache/james/util/OptionalUtils.java

```diff
@@ -45,4 +45,17 @@
             .filter(Optional::isPresent)
             .orElse(optional2);
     }
+
+    public static <T> boolean containsDifferent(Optional<T> requestValue, T storeValue) {
+        return requestValue
+            .filter(value -> !value.equals(storeValue))
+            .isPresent();
+    }
+
+    public static <T> boolean requestChanged(Optional<T> requestValue, Optional<T> storeValue) {
```

Comment:
I don't understand what mean "requestChanged" in a context of OptionalUtils

Reply:
How is about "contentChangedCompareToStoreValue"? I changed to "contentChangedCompareToStoreValue"